### PR TITLE
CLI: refactor

### PR DESCRIPTION
**Pull request by vaijab**
Mainly, moved resources into separate subcommands for better UX and
simpler code.

### CODE REVIEW

File: pkg/keto/cmd/get.go [hunk 1]
```diff
@@ -17,71 +17,106 @@ limitations under the License.
...
 	}
-	res := args[0]
-	resName := ""
-	if len(args) == 2 {
-		resName = args[1]
+
+	name := ""
+	if len(args) == 1 {
+		name = args[0]
 	}
```
Comment (Joseph-Irving) [inline pkg/keto/cmd/get.go]: Is there ever a case where len(args) will be >1? Would it not error before this if you had two arguments for this command?

Comment (vaijab) [inline pkg/keto/cmd/get.go]: Normally there should not be, but if a user enters 2 args, what should be the behaviour?

Comment (vaijab) [inline pkg/keto/cmd/get.go]: The idea is that if you don't provide a specific resource name, then it will return all resources of a given type.

Comment (Joseph-Irving) [inline pkg/keto/cmd/get.go]: Well potentially you could have someone request info about two compute pools e.g. `keto get computepool joepool1 joepool2` in which you would expect it to return info for those two specifically.

Comment (vaijab) [inline pkg/keto/cmd/get.go]: > you could have someone request info about two compute pools

Yes, that is something we'll have to build and would definitely +1 that.

**Pull request merged**